### ISSUE TITLE
FTP issue

### ISSUE THREAD

**Issue by P4risAndStuff** [ShareX/ShareX#2455]
Hi jaex, sorry for the last issue - apologies to you. I gotta own up to my mistakes and stop messing around.
Here's my issue:
![](https://vgy.me/Of2w0j.png)

essentially, i'm trying to get files to upload to my website so i configured FTP in destinations, and then i set "file uploads" to FTP in the home window for sharex. i *think* i did everything right, but for some reason, the files are not uploading to my website. i tried to upload a test file, i got a 404 when i went to the URL that it gave me after the upload finished.

i know what my info for the FTP is, i entered it correctly. i used the same info on filezilla and it worked there. what should i do? did i do something wrong? are there any advanced FTP options i can try?

**Comment by Jaex** [comment]
Did you check with FileZilla is files uploaded correctly from ShareX?

**Comment by P4risAndStuff** [comment]
@Jaex I think I figured it out actually. I had to change the value for SubFolderPath to "htdocs/files" instead of just "files" because "htdocs" is the name of the folder in my websites files that contains all the publicly accessible files on my website. So after I changed it to that, I tried uploading another test file, this time it was a random small pdf file on my computer, and it uploaded! I'm not sure if it only uploaded  this time because of how it was a small file and/or a pdf file, but it worked this time so I'll try uploading other files.
